internal/collect/inband.go

```diff
@@ -53,11 +53,16 @@ func (i *InbandCollector) InventoryLocal(ctx context.Context) (*model.Asset, err
 		return nil, err
 	}
 
+	biosConfig, err := i.deviceManager.GetBIOSConfiguration(ctx)
```

Comment:
Can you add a few comments to this parent method and the interface that the Inventory also collects BIOS configuration.

Reply:
Fixed in [a885fbb](https://github.com/metal-toolbox/alloy/pull/83/commits/a885fbb9f131812540554ef12f9048d3a194b4bd)